openupgradelib/openupgrade.py

```diff
@@ -3492,3 +3493,31 @@ def cow_templates_replicate_upstream(cr, mark_colname=None):
         """
         ).format(mark_identifier),
     )
+
+
+def clean_transient_models(cr):
+    """Clean transient models to prevent possible issues due to
+    chained data.
+
+    To be run at the base post-migration script for having a general scope.
+    Only works on > v8.
+
+    :param cr: Database cursor.
+    """
+    if version_info[0] < 9:
+        raise Exception("Not supported Odoo version for this method.")
+    cr.execute("SELECT model FROM ir_model WHERE transient")
+    table_names = [x[0] for x in cr.fetchall()]
```

Comment:
Please replace dots by underscores and skip if the resulting table name does not exist in the database.

Reply:
Done